### ISSUE TITLE
[Feature Request] new tab at end

### ISSUE THREAD

**Issue by lonix1** [autonome/Always-Right#26]
Like in extension: https://addons.mozilla.org/en-US/firefox/addon/open-tabs-next-to-current/

<kbd>ctrl+y</kbd> opens a new tab at the very end

**Comment by autonome** [comment]
I agree it would be nice to have an escape hatch for when you want to have a new tab opened at the end of the tab strip.

I would prefer a modifier to the new tab shortcut though, instead of a letter combo.
